### ISSUE TITLE
Not compatible with Python 3.7

### ISSUE THREAD

**Issue by srcr** [countercept/snake-tail#1]
From my investigation it looks like snake-tail isn't compatible with Python 3.7 due to the use of the async option, this feature changed for Pyhton 3.7 

```
Traceback (most recent call last):
  File "~/.local/bin//snake-tail", line 11, in <module>
    load_entry_point('snake-tail==1.0', 'console_scripts', 'snake-tail')()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "~/.local/lib/python3.7/site-packages/snake_tail/snake_tail.py", line 122
    asynchronous=args.async,
```

**Comment by alexkornitzer** [comment]
Good spot, this is probably needs some other updates too, as you might be able to tell we don't use it much. I will get it back up to speed in the next few days.

**Comment by alexkornitzer** [comment]
Now fixed (or should be) in version 1.0.1.
